--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -53,6 +53,8 @@ endif()
 set(HPIPM_TARGET "X64_AUTOMATIC" CACHE STRING "HPIPM Target architecture")
 set(LA "HIGH_PERFORMANCE" CACHE STRING "Linear algebra optimization level")

+set(CMAKE_SYSTEM_NAME "Linux" CACHE STRING "System name")
+
 if (CMAKE_SYSTEM_NAME MATCHES "Windows")
     set(BUILD_SHARED_LIBS OFF CACHE STRING "Build shared libraries")
 else()