--- a/msgq/ipc.cc
+++ b/msgq/ipc.cc
@@ -13,14 +13,7 @@ const bool MUST_USE_ZMQ = false;
 #endif

 bool messaging_use_zmq(){
-  if (std::getenv("ZMQ") || MUST_USE_ZMQ) {
-    if (std::getenv("OPENPILOT_PREFIX")) {
-      std::cerr << "OPENPILOT_PREFIX not supported with ZMQ backend\n";
-      assert(false);
-    }
-    return true;
-  }
-  return false;
+  return true;
 }

 bool messaging_use_fake(){